packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -293,6 +296,9 @@ const _generateStyles = (
                 borderColor: tokens.color.offBlack16,
                 color: tokens.color.offBlack64,
                 cursor: "auto",
+                ":focus": {
```

Comment:
**suggestion/issue:** This probably needs to be changed to `:focus-visible`. Right now, if we use `focus`, then the disabled state will also apply when the user presses on the button (via mouse for example), and the would misguide the user as it would think that the button is interactive.

Reply:
That said, I understand if you want to leave this fix for the next PR (as you mentioned in the description :)).

Reply:
Good catch! I will update this in my next PR that makes sure all the states are correct!